### PR TITLE
And/pe 128surveymonkey

### PR DESCRIPTION
**Pull request by andrey-canon**
## Description 
This branch contains the first version for SurveyMonkeyXblock, the files added are the following:
- surveymonkey.py -> Main file, variables methods and logic
- surveymonkey.html -> Html to be shown 
- surveymonkey.css-> Styles" Empty for now "
- surveymonkey.js -> Javascript code " Empty for now "
- setup.py -> contains version and requirements to run the xblock
- django.po -> The string to be translate (Only for french)

## Reviewers 
- [ ] @diegomillan 
- [ ] @ericfab179

### CODE REVIEW

File: surveymonkey/surveymonkey.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
+"""TO-DO: Write a description of what this XBlock is."""
```
Comment (ericfab179) [inline surveymonkey/surveymonkey.py]: Please complete docstring.

**Comment by andrey-canon** [comment]
@ericfab179 I have add the docstring.
@diegomillan  Please make the review

**Review by ericfab179 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1 +1,41 @@
...
+-   I can set in a SurveyMonkey-XBlock a survey URL.
+-   I can select to make the survey trackable (i.e. pass user id across via URL) - or not.
+-   I can add some introductory text which is displayed to the student above the survey link.
+-   I can define the text used to link to the survey (or leave it blank to use the default which should be “complete the survey”).
+-   I can define the text used in the xblock title
+
+
+### Student view criteria
+-   Clicking the survey link should open the survey in a new browser tab.
+-   If user tracking is selected, then the survey URL includes a `uuser_anonymous_id` custom var with the student's anonymous user id
```
Comment (diegomillan) [inline README.md]: Minor fix `user_anonymous_id`

File: README.md [hunk 3]
```diff
@@ -1 +1,41 @@
...
+-   I can add a “SurveyMonkey” Xblock.
+-   I can set in a SurveyMonkey-XBlock a survey URL.
+-   I can select to make the survey trackable (i.e. pass user id across via URL) - or not.
+-   I can add some introductory text which is displayed to the student above the survey link.
+-   I can define the text used to link to the survey (or leave it blank to use the default which should be “complete the survey”).
+-   I can define the text used in the xblock title
+
+
+### Student view criteria
+-   Clicking the survey link should open the survey in a new browser tab.
```
Comment (diegomillan) [inline README.md]: Should say: Clicking over the link opens the survey in a new browser tab

File: README.md [hunk 4]
```diff
@@ -1 +1,41 @@
...
+* Click the "Save changes" button.
+
+## Usage
+
+### Course Author criteria
+-   I can add a “SurveyMonkey” Xblock.
+-   I can set in a SurveyMonkey-XBlock a survey URL.
+-   I can select to make the survey trackable (i.e. pass user id across via URL) - or not.
+-   I can add some introductory text which is displayed to the student above the survey link.
+-   I can define the text used to link to the survey (or leave it blank to use the default which should be “complete the survey”).
```
Comment (diegomillan) [inline README.md]: letś just say  "I  can define the text used to link to the survey (or leave it blank to use the default)"

File: README.md [hunk 5]
```diff
@@ -1 +1,41 @@
...
+* From the main page of a specific course, navigate to `Settings -> Advanced Settings` from the top menu.
+* Check for the `Advanced Module List` policy key, and Add ``"surveymonkey"`` to the policy value list.
+* Click the "Save changes" button.
+
+## Usage
+
+### Course Author criteria
+-   I can add a “SurveyMonkey” Xblock.
+-   I can set in a SurveyMonkey-XBlock a survey URL.
+-   I can select to make the survey trackable (i.e. pass user id across via URL) - or not.
```
Comment (diegomillan) [inline README.md]: Please, Use choose instead of select

File: README.md [hunk 6]
```diff
@@ -1 +1,41 @@
-# openedx-surveymonkey-xblock
+# Xblock Openedx-Surveymonkey
+
+The Openedx-Surveymonkey Xblock allows to embed a survey link inside a unit, also, allows to include the survey description and title. The xblock allows you to include the user anonymous id as a custom variable,  that is a logic feature that allows you to track data about respondents by passing one or more values through a survey link and into your survey results,.
```
Comment (diegomillan) [inline README.md]: extra `,` after results.

**Review by diegomillan — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (andrey-canon) [reply README.md]: ok

File: README.md (hunk 3 above)
Comment (andrey-canon) [reply README.md]: ok

File: README.md (hunk 4 above)
Comment (andrey-canon) [reply README.md]: ok

File: README.md (hunk 5 above)
Comment (andrey-canon) [reply README.md]: ok

File: README.md (hunk 6 above)
Comment (andrey-canon) [reply README.md]: ok

**Comment by andrey-canon** [comment]
@diegomillan requested changes has been amended

**Review by diegomillan — APPROVED** [review]

**Pull request merged**